docs/odh/oci-model-storage.md

```diff
@@ -0,0 +1,208 @@
+# Using OCI containers for model storage
+
+Starting ODH 2.18, the ability to use OCI containers as storage for models
+is enabled in KServe by default. The benefits of using OCI containers for
+model storage are described in [the upstream KServe project
+documentation](https://kserve.github.io/website/latest/modelserving/storage/oci/),
+which also describes how to deploy models from OCI images. 
+
+This page offers a guide similar to the upstream project documentation, but
+focusing on the OpenDataHub and OpenShift characteristics. To demonstrate
+how to create an OCI image, the publicly available [MobileNet v2-7
+model](https://github.com/onnx/models/tree/main/validated/vision/classification/mobilenet)
+is used. This model is in ONNX format.
+
+The ODH projects provides configurations for the OpenVINO model server, which
+supports models in ONNX format. Thus, this guide will use this model server
+to demonstrate how deploy the MobileNet v2-7 model stored in an OCI image.
+
+## Storing a model in an OCI image
+
+Start by creating an empty directory for downloading the model and creating
+the necessary support files to create the OCI image. You may use a temporary
+directory by running the following command:
+```shell
+cd $(mktemp -d)
+```
+
+OpenVINO expects a specific directory tree for model versioning.
+Starting from some base directory, its contents should be a collection of
+numbered subdirectories using positive integer values. The numbers would
+represent the versions of the model. When using OCI images, this
+structure may be irrelevant, as you can use the OCI container registry
+features. However, since OpenVINO expects the versioned directory structure, a
```

Comment:
what features are you referring to?

Reply:
For example, typically image tags are used for versions.
But, ultimately, it depends how the user is using image repositories and tags. So, I didn't want to mention anything.